test/TesterInternal/ConfigTests.cs

```diff
@@ -833,7 +838,8 @@ public void ServerConfig_LoadFrom()
 
             Assert.Equal(filename, Path.GetFileName(config.SourceFile)); // OrleansConfiguration.SourceFile
         }
-
+#if !NETSTANDARD_TODO
+        //blocked by cannot update System.Xml.XmlDocument to 4.0.1 which is compatible with .net framework 4.6
```

Comment:
hmmm, very strange. Let's see if we can quickly fix the dependency instead of marking a lot of related parts of the code with TODO